hoot-services/src/main/java/hoot/services/HootProperties.java

```diff
@@ -224,9 +220,16 @@
         SEED_RANDOM_QUERIES = getProperty("seedRandomQueries");
         INTERNAL_JOB_THREAD_SIZE = getProperty("internalJobThreadSize");
         TRANSLATION_SCRIPT_PATH = getProperty("translationScriptPath");
-        UPLOAD_FOLDER = HOME_FOLDER + File.separator + "upload";
         DERIVE_CHANGESET_SCRIPT = getProperty("deriveChangesetScript");
 
+        USERFILES_FOLDER = HOME_FOLDER + File.separator + "userfiles";
+        SCRIPT_FOLDER = USERFILES_FOLDER + File.separator + "customscript";
+        TEMP_OUTPUT_PATH = USERFILES_FOLDER + File.separator + "tmp";
+        UPLOAD_FOLDER = USERFILES_FOLDER + File.separator + "upload";
+        RPT_STORE_PATH = USERFILES_FOLDER + File.separator + "reports";
+        BASEMAPS_TILES_FOLDER = USERFILES_FOLDER + File.separator + "ingest" + File.separator + "processed" + File.separator + "BASEMAP";
+        BASEMAPS_FOLDER = USERFILES_FOLDER + File.separator + "ingest" + File.separator + "upload" + File.separator + "BASEMAP";
+
```

Comment:
I doubt that anyone ever touches these properties.  In fact, these locations are hard coded in the RPMs.  So, I thought that if we made them "immutable", things would be more predictable.  Also less clutter in hoot-services.conf.in